app/javascript/mastodon/features/explore/index.js

```diff
@@ -1,18 +1,18 @@
-import React from 'react';
-import { defineMessages, injectIntl, FormattedMessage } from 'react-intl';
-import { connect } from 'react-redux';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import { FormattedMessage, defineMessages, injectIntl } from 'react-intl';
+import { connect } from 'react-redux';
+import { NavLink, Route, Switch } from 'react-router-dom';
 import Column from 'mastodon/components/column';
 import ColumnHeader from 'mastodon/components/column_header';
-import { NavLink, Switch, Route } from 'react-router-dom';
+import Search from 'mastodon/features/compose/containers/search_container';
+import { showTrends } from 'mastodon/initial_state';
 import Links from './links';
-import Tags from './tags';
+import SearchResults from './results';
 import Statuses from './statuses';
 import Suggestions from './suggestions';
-import Search from 'mastodon/features/compose/containers/search_container';
-import SearchResults from './results';
-import { Helmet } from 'react-helmet';
-import { showTrends } from 'mastodon/initial_state';
+import Tags from './tags';
```

Comment:
Could you not reorder all of that? :sweat_smile:

Reply:
dd66c9850bbdad98da6618224a096bdd828a6a5a

You are right, it would be better to keep them separate. I wanted to address this collectively at a later date by adding ESLint rules governing the order of imports.